core/geometry/src/test/polyface/Polyface.test.ts

```diff
@@ -1672,44 +1674,38 @@ function createPolyfaceFromSynchroA(geom: any): Polyface {
   const polyface = IndexedPolyface.create(hasNormals, hasUV);
   // const numVertex = geom.vertex_array.length;
 
-  // Always load point data ...
+  // always load point data
   for (const xyz of geom.vertex_array)
     polyface.data.point.pushXYZ(xyz.x, xyz.y, xyz.z);
-
   // OPTIONAL uv array
   if (hasUV) {
     for (const xy of geom.uv_array)
       polyface.data.param!.pushXY(xy.x, xy.y);
   }
-
   // OPTIONAL normal array
   if (hasNormals) {
     for (const xyz of geom.normal_array)
       polyface.data.normal!.pushXYZ(xyz.x, xyz.y, xyz.z);
   }
 
-  // build up the index sets one facet at a time, let terminateFacet () get the facetStart table right . . .
-  // Note that polyface has separate index arrays for params and normals.
-  // (When there are sharp edges in the mesh, this allows smaller param, point, and normal arrays.
-  //    The single-index style of Synchro and other
-  //   browser software saves memory by sharing the index array but wastes it with redundant data arrays.
-  //   The polyface compress function can get this effect)
+  // build up the index sets one facet at a time, let terminateFacet () get the facetStart table right
+  // note that polyface has separate index arrays for params and normals.
+  // (when there are sharp edges in the mesh, this allows smaller param, point, and normal arrays.
+  // the single-index style of Synchro and other
+  // browser software saves memory by sharing the index array but wastes it with redundant data arrays.
+  //  the polyface compress function can get this effect)
```

Comment:
When the comments are paragraphs, prefer proper sentence punctuation and capitalization.

```suggestion
  // Build up the index sets one facet at a time: let terminateFacet() get the facetStart table right.
  // Note that polyface has separate index arrays for params and normals (when there are sharp
  // edges in the mesh, this allows smaller param, point, and normal arrays.
  // The single-index style of Synchro and other browser software saves memory by sharing the
  // index array but wastes it with redundant data arrays.
  // The polyface compress function can remove this effect.
```

Reply:
For consistency I start all comments inside the body with lower case.

Reply:
This makes multi-line comments harder to read.

Reply:
ok